src/main/java/com/schematica/world/storage/Schematic.java

```diff
@@ -73,19 +75,21 @@ private boolean hasOffset()
 
     //MINECOLONIES END
 
+    @SuppressWarnings("deprecation")
     @Override
     public IBlockState getBlockState(@NotNull final BlockPos pos)
     {
         if (isInvalid(pos))
         {
-            return Blocks.air.getDefaultState();
+            return Blocks.AIR.getDefaultState();
         }
 
         final int x = pos.getX();
         final int y = pos.getY();
         final int z = pos.getZ();
         final Block block = BLOCK_REGISTRY.getObjectById(this.blocks[x][y][z]);
 
+        //todo: be aware of deprecation and possible removal
         return block.getStateFromMeta(this.metadata[x][y][z]);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "getStateFromMeta"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)